packages/wrangler/src/vectorize/client.ts

```diff
@@ -97,29 +98,37 @@ export async function updateIndex(
 export async function insertIntoIndex(
 	config: Config,
 	indexName: string,
-	vectors: Array<VectorizeVector>
+	body: FormData
```

Comment:
Changed this to always assume multipart and file source. I can't really see a reality where customers specify vector definitions via a flag